examples/index.json

```diff
@@ -1071,5 +1067,15 @@
                 "url": "table-queries"
             }
         ]
+    },
+    {
+        "title": "Lock",
```

Comment:
Shall we move this section to just after L1008 so that everything in column 0 are together?